framework/src/modules/chain/logic/block.js

```diff
@@ -251,9 +253,7 @@ class Block {
 				throw invalidTransactionResponse.errors;
 			}
 		} catch (errors) {
-			const error =
-				Array.isArray(errors) && errors.length > 0 ? errors[0] : errors;
-			throw error;
+			throw errors;
```

Comment:
Are the items contained in `errors` Error instances?

Reply:
Yes or more specifically `TransactionError` from elements